jms/connection.py

```diff
@@ -0,0 +1,43 @@
+"""Mini framework for JMS interaction"""
+import stomp
+from logger_config import get_logger
+
+logger = get_logger(__name__)
+
+
+class JMSConnection:
+    """Establish a JMS connection using the stomp protocol"""
+
+    def __init__(self):
+        self.host = None
+        self.port = None
+        self.login = None
+        self.password = None
+        self.connection = None
+
+    def connect(self, host, port, login, password):
```

Comment:
It would be better to have those parameters passed to `__init__` instead of `connect`